src/webparts/controlsTest/components/ControlsTest.tsx

```diff
@@ -1,197 +1,218 @@
-import * as React from "react";
-import {
-  IBasePickerStyles,
-  ITag,
-} from "office-ui-fabric-react/lib/Pickers";
-import {
-  Stack,
-} from "office-ui-fabric-react/lib/Stack";
-import {
-  Text,
-} from "office-ui-fabric-react/lib/Text";
-import {
-  TextField
-} from "office-ui-fabric-react/lib/TextField";
+import * as React from 'react';
```

Comment:
Can you try to revert the formatting changes?

Reply:
done